tests/utils.go

```diff
@@ -360,16 +327,27 @@ func newPVC(os string, withAuth bool) *k8sv1.PersistentVolumeClaim {
 	}
 }
 
-func newPV(os string, lun int32, withAuth bool) *k8sv1.PersistentVolume {
+func createPvISCSI(os string, lun int32, withAuth bool) {
```

Comment:
If we drop ISCSI volume source support we can remove all functions related to it

Reply:
This function is creating the PV for the iscsi demo target. All the direct LUN related functions have been removed now.